app/apps/client/admin/helpers.js

```diff
@@ -357,15 +357,19 @@ export const appStatusSpanProps = ({
 export const formatPrice = (price) => `\$${ Number.parseFloat(price).toFixed(2) }`;
 
 export const formatPricingPlan = (pricingPlan) => {
-	const perUser = pricingPlan.isPerSeat && pricingPlan.tiers && pricingPlan.tiers.length;
+	const { strategy, price } = pricingPlan;
+
+	const tier = Array.isArray(pricingPlan.tiers)
+		? pricingPlan.tiers.find((tier) => tier.price === price)
+		: null;
 
 	const pricingPlanTranslationString = [
 		'Apps_Marketplace_pricingPlan',
-		pricingPlan.strategy,
-		perUser && 'perUser',
+		strategy,
+		(tier && tier.perUnit) && 'perUser',
```

Comment:
```suggestion
		tier && tier.perUnit && 'perUser',
```

Reply:
I thought our lint wouldn't allow this :thinking: or I'm missing something =P

Reply:
Either it does or `--no-verify` 😬